src/HtmlGenerator/Program.cs

```diff
@@ -101,12 +138,18 @@ private static void Main(string[] args)
             Log.MessageLogFilePath = Path.Combine(Paths.SolutionDestinationFolder, Log.MessageLogFile);
 
             // Warning, this will delete and recreate your destination folder
-            Paths.PrepareDestinationFolder();
+            Paths.PrepareDestinationFolder(force);
 
             using (Disposable.Timing("Generating website"))
             {
-                IndexSolutions(projects, properties);
-                FinalizeProjects(emitAssemblyList);
+                var federation = noBuiltInFederations ? new Federation(null) : new Federation();

```

Comment:
yes, that would work I guess. I am not at a pc to try right now. I will look into it later if you like.